tests/common/beamlines/test_device_instantiation.py

```diff
@@ -31,6 +69,29 @@ def test_device_creation(RE, module_and_devices_for_beamline):
     assert len(beamline_utils.ACTIVE_DEVICES) == len(devices)
 
 
+@pytest.mark.parametrize(
```

Comment:
I'd make a special p38 test that we can add to when we have additional lab beamlines

Reply:
it's functionally equivalent...

Reply:
As long as the tests cover both cases I'm not bothered at the moment, can revisit later